Assets/Scripts/tools/Freeform.cs

```diff
@@ -933,15 +933,18 @@ private void AddCheckpoint()
                 // Create the new 'side faces' with clockwise ordering. Each side face has 4 verts.
                 // The first face 'loops around'.
                 List<int> newFaceVertexIds = new List<int>() {
-            currentFrontFace.vertexIds[i-1],
-            currentFrontFace.vertexIds[i % vertexCount],
-            newVertices[i % vertexCount].id,
-            newVertices[i-1].id
-          };
+					currentFrontFace.vertexIds[i-1],
```

Comment:
We use `dotnet format` to do some code formatting, and this change wasn't so well liked :-)

Can I ask you to please undo  this part, and just keep the actual change you made on line 945?

Reply:
that said, I'm not sure *why*; it looks like the formatting here (in our original code) was wrong! But we'll fix that later in a separate PR.

If you're not sure how to undo this change, that's cool -- just let me know, and if you want, I can push a commit to your branch that will do it.

Reply:
Yes, was meant as a fix for the format, have my IDE on tabs. I think I changed it back now and used spaces everywhere.

Reply:
Thanks! We have an .editorconfig which many, but not all, programs respect; if you know if there's something we can add that your IDE would like, we'll be glad to do so